packages/superset-ui-core/src/query/extractExtras.ts

```diff
@@ -6,6 +6,7 @@ export default function extractExtras(formData: QueryFormData): Partial<QueryObj
   const partialQueryObject: Partial<QueryObject> = {
     filters: formData.filters || [],
     extras: formData.extras || {},
+    applied_time_extras: {},
   };
 
   const reservedColumnsToQueryField: Record<string, keyof QueryObject> = {
```

Comment:
Looks like this could be `Record<QueryObjectAppliedTimeExtraKey, keyof QueryObject>` or `Record<TimeColumnConfigKey, keyof QueryObject>`.